File: bookhub-back/src/main/java/com/example/bookhub_back/dto/stock/response/StockResponseDto.java
package com.example.bookhub_back.dto.stock.response;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class StockResponseDto {
    private Long stockId;
    private String type;
    private Long branchId;
    private String branchName;
    private String bookIsbn;
    private String bookTitle;
    private Long amount;
    private Long bookAmount;
}

File: bookhub-front/src/dtos/stock/Stock.response.dto.ts
import { StockActionType } from "@/apis/enums/StockActionType";

export interface StockResponseDto{
    stockId: number;
    branchId: number;
    isbn : string;
    branchName: string;
    bookTitle: string;
    amount : number;
}

export interface StockUpdateResponseDto{
    stockId: number;
    branchName: string;
    type: StockActionType;
    bookTitle: string;
    amount: number;
    bookAmount: number;
}